runtime/v2/shim/shim_unix.go

```diff
@@ -1,3 +1,4 @@
+//go:build !windows
```

Comment:
These seem to not have been part of the original commit

Reply:
aah! these are added when I did `go fmt ./...`

Reply:
Yes, I suspected that happened. Not a huge deal I guess (but perhaps cleaner to do them all at once if we decide to update this branch to go1.17+)

Reply:
Removing them fixed the issue.